tests/MiddlewareResolverTest.php

```diff
@@ -274,14 +273,6 @@ public function testResolveClassNameHandlerWithDependenciesPassingContainerWitho
         $middleware->process($this->request, $this->handler);
     }
 
-    public function testResolveClassNameHandlerWithDependenciesPassingContainerWithAutoWiringToConstructor(): void
-    {
-        $resolver = new MiddlewareResolver(new Container());
-        $middleware = $resolver->resolve(RequestHandlerAutoWiring::class);
-        $this->assertInstanceOf(MiddlewareInterface::class, $middleware);
-        $this->assertInstanceOf(ResponseInterface::class, $middleware->process($this->request, $this->handler));
-    }
-
```

Comment:
Why was this test case deleted?

Reply:
I already wrote about it at the beginning
I had to remove one test method that is not related to httpsoft/http-runner, which tested devanych/di-container auto-wiring feature and interfered with the use psr/container 2.0. The end user may well use another container that does not have auto-wiring functionality (for example laminas/servicemanager), therefore, it doesn't make sense to test it here.

Reply:
Well, or if you leave this test, then you will have to update the `devanych/di-container` package so that it allows psr/container 2.0

Reply:
> I already wrote about it at the beginning I had to remove one test method that is not related to `httpsoft/http-runner`, which tested `devanych/di-container` auto-wiring feature and interfered with the use psr/container 2.0. The end user may well use another container that does not have auto-wiring functionality (for example `laminas/servicemanager`), therefore, it doesn't make sense to test it here.

Sorry. I missed :)

Reply:
> Well, or if you leave this test, then you will have to update the [devanych/di-container](https://github.com/devanych/di-container/blob/master/composer.json#L21) package so that it allows psr/container 2.0

You can leave test case. I have updated `devanych/di-container`.